src/schema/project-fund.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  char,
+  numeric,
+  pgEnum,
+  pgTable,
+  text,
+  uuid,
+} from "drizzle-orm/pg-core";
+import { projectCategoryEnum, projectFundStageEnum } from "./project";
+import { z } from "zod";
+import { managingCodeEntitySchema } from "./managing-code";
+
+export const fundCategoryEnum = pgEnum("fund_category", [
+  "city-non-exempt",
+  "city-exempt",
+  "city-cost",
+  "non-city-state",
+  "non-city-federal",
+  "non-city-other",
+  "non-city-cost",
+  "total",
+]);
+
+export const projectFund = pgTable("project_fund", {
+  id: uuid("id"),
+  managingCode: char("managing_code", { length: 3 }),
+  projectId: text("project_id"),
+  category: projectCategoryEnum("project_category"),
+  stage: projectFundStageEnum("project_fund_stage"),
```

Comment:
We'll also want a 'multicolumn' foreign key here with 'managing_code' and 'project_id'